chemprop/data/scaffold.py

```diff
@@ -70,7 +70,9 @@ def scaffold_split(data: MoleculeDataset,
     :return: A tuple of :class:`~chemprop.data.MoleculeDataset`\ s containing the train,
              validation, and test splits of the data.
     """
-    assert sum(sizes) == 1
+    #TODO(degraff): floating point math could mess this up
+    if sum(sizes) != 1:
```

Comment:
Perhaps we could do `np.isclose(sum(sizes),1)` to avoid the potential floating point math problem you point out here?